wordpress-mobile/WordPress-Android
# Add tracking event for Plan details

Fixes #12083

This PR adds a tracking event when a user taps on specific plan in the Plans page.

## To test

0. Enable Collect information via Me -> App Settings -> Privacy Settings
1. Click Plan menu on the My site tab -> Plans page is opened
2. Click any plan on the Plans page
3. Notice that the console contains the following log:
`🔵 Tracked: plans_compare`

## Reviewing

Only 1 reviewer is needed but anyone can review.

PR submission checklist:
